src/SFA.DAS.Employer.PR.Web/Controllers/ProvidersController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Employer.PR.Domain.Interfaces;
+using SFA.DAS.Employer.PR.Domain.Models;
+using SFA.DAS.Employer.PR.Web.Infrastructure;
+using SFA.DAS.Employer.PR.Web.Services;
+
+namespace SFA.DAS.Employer.PR.Web.Controllers;
+
+public class ProvidersController(IOuterApiClient outerApiClient, ICacheStorageService cacheStorageService) : Controller
+{
+    [HttpGet]
+    [Route("/providers")]
```

Comment:
I think we better rename this to `registeredproviders`

Reply:
done